en/docs/fragments/pickup-manager-saml.md

```diff
@@ -6,8 +6,7 @@
 
 2. Click **Service Providers >Add** and enter "pickup-dispatch" as the **Service Provider Name** and click **Register**.
```

Comment:
A space after > ?

Shall we write "click Register" as a separate sentence in the same step?